frontend/src/features/public-form/utils/inputValidation.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  CheckboxFieldValues,
+  MultiAnswerValue,
+  RadioFieldValues,
+  SingleAnswerValue,
+  TableFieldValues,
+  VerifiableFieldValues,
+} from '~templates/Field/types'
+
+/**
+ * Regex to validate yyyy-mm-dd strings.
+ * @note this does not validate that the date is valid, just that the format is correct.
+ */
+const ISO_8601_DATE_ONLY_REGEX =
+  /^\d{4}-([0][1-9]|1[0-2])-([0-2][1-9]|[1-3]0|3[01])$/
```

Comment:
If this regex was taken from somewhere else, could we add the source as a comment, just for future reference please

Reply:
`inputValidation` util deleted.